lib/workers/repository/dependency-dashboard.ts

```diff
@@ -38,16 +38,24 @@ const markedBranchesRe = regEx(
   'g',
 );
 
+const approveAllPendingPrs = 'approve-all-pending-prs';
+const createAllRateLimitedPrs = 'create-all-rate-limited-prs';
+const rebaseAllOpenPrs = 'rebase-all-open-prs';
+
+function isCheckboxMarked(issueBody: string, type: string): boolean {
```

Comment:
maybe extract this refactor to merge first?

Reply:
I created #37339, and will create another PR for the other refactoring of this class regarding the generation of branch lists